modules/ding_borgun_checkout/ding_borgun_checkout.admin.inc

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * @file
+ * Administrative functions for Ding Borgun.Checkout.
+ */
+
+/**
+ * Settings form.
+ */
+function ding_borgun_checkout_settings($form, $form_state) {
+  $form['ding_borgun_checkout_host'] = array(
+    '#type' => 'radios',
+    '#title' => t('Use evironment'),
+    '#description' => t('Whether to use the test evironment at Borgun, or the live environment.'),
+    '#options' => array(
+      'https://test.borgun.is' => t('Test'),
+      'https://ecommerce.borgun.is' => t('Live'),
+    ),
+    '#default_value' => variable_get('ding_borgun_checkout_host', 'https://test.borgun.is'),
+  );
+
+  $form['ding_borgun_checkout_private_token'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Private token'),
+    '#description' => t('Private token provided by Borgun.'),
+    '#default_value' => variable_get('ding_borgun_checkout_private_token', ''),
```

Comment:
None of these fields are required. Is that intentional?

Reply:
Nope, fixed